flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/utils/JavaUserDefinedTableFunctions.java

```diff
@@ -166,4 +169,91 @@ public void eval(String str) {
                             str.length() + 5));
         }
     }
+
+    /** Mutable accumulator of structured type for the table aggregate function. */
```

Comment:
nit: add class JavaUserDefinedTableAggFunctions.java to restore these function (+1), or just using UserDefinedTableAggFunctions.scala.